src/lib/validator-select_multiple.ts

```diff
@@ -0,0 +1,35 @@
+import {ContactProperty} from '../config';
+import {IValidator} from './validation';
+
+export default class ValidatorSelectMultiple implements IValidator {
+  isValid(input: string, property: ContactProperty): boolean {
+    // Verify property.parameter is an object and is not null
+    if (!property?.parameter || typeof property.parameter !== 'object') {
```

Comment:
haven't through this through - but could this just use `ValidatorSelectOne.isValid()` and pass each value in the input one by one?